### PR TITLE
US-3.4.2: Update perk store widget

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: digital-workplace-webapps/src/main/webapp/vue-app/perkstore/components/Perkstore.vue [hunk 1]
```diff
@@ -15,23 +15,7 @@
...
-                  width="20"
-                  class="mb-1"
-                  v-on="on">
-                  <a 
-                    class="white--text caption" 
-                    style="padding-right: 1px" 
-                    @click="navigateTo('perkstore?notProcessedOrders=true')">{{ pendingOrdersSize }}</a>
-                </v-btn>
-              </a>
+              <span class="subtitle-2 grey--text"><a class="grey--text" @click="navigateTo('perkstore?notProcessedOrders=true')">{{ this.$t('homepage.Perkstore.myOrders') }}</a></span>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/perkstore/components/Perkstore.vue]: Put the @click on the whole element

Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/perkstore/components/Perkstore.vue]: Done

File: digital-workplace-webapps/src/main/webapp/vue-app/perkstore/components/Perkstore.vue [hunk 2]
```diff
@@ -10,28 +10,13 @@
...
-                  width="20"
-                  class="mb-1"
-                  v-on="on">
-                  <a 
-                    class="white--text caption" 
-                    style="padding-right: 1px" 
-                    @click="navigateTo('perkstore?notProcessedOrders=true')">{{ pendingOrdersSize }}</a>
-                </v-btn>
-              </a>
+              <span class="subtitle-2 grey--text"><a class="grey--text">{{ this.$t('homepage.Perkstore.myOrders') }}</a></span>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/perkstore/components/Perkstore.vue]: the <a></a> not needed, cursor:pointer on the whole block will be sufficient

Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/perkstore/components/Perkstore.vue]: Done

**Pull request merged**